sqlmesh/core/config/connection.py

```diff
@@ -1672,6 +1672,48 @@ def get_catalog(self) -> t.Optional[str]:
         return self.catalog_name
 
 
+class RisingwaveConnectionConfig(ConnectionConfig):
```

Comment:
Same here. If there is nothing different in the connection properties between `RisingWaveConnectionConfig` and the `PostgresConnectionConfig`, why not just extend `PostgresConnectionConfig` and override the internal properties like `type_` and `_engine_adapter` etc instead of copy+paste?

Unnecessary copy+paste is considered a code smell and doesn't meet the standards of this codebase